package.json

```diff
@@ -5,12 +5,20 @@
   "description": "Bloomberg Open API (BLPAPI) binding for node.js",
   "os": [ "linux", "win32", "darwin" ],
   "cpu": [ "x64", "ia32" ],
+  "dependencies": {
+    "bluebird": "^2.9.13",
+    "custom-error-generator": "7.0.0",
+    "debug": "^2.1.0",
+    "lodash": "^2.4.1"
+  },
+  "devDependencies": {
+    "tslint": "^2.1.1",
+    "typescript": "^1.4.1"
+  },
   "scripts": {
+    "prepublish": "make",
     "install": "node-gyp configure build"
```

Comment:
We should look at `npm help scripts` and update the `Makefile` and this section appropriately.  It would be great to capture the `node-gyp` build in the `Makefile`.